### DISCUSSION TITLE
Add styled components examples

### DISCUSSION THREAD

**Discussion by abdulrahimiliasu** [radix-ui/website#625] [Ideas]
It would be nice to add `styled-components` usage examples too

**Comment by asherccohen** [comment]
I can easily provide styled-components versions if the team is interested in showcasing them.

**Reply by abdulrahimiliasu** [reply]
That would be awesome

**Reply by asherccohen** [reply]
Should I open PRs in the docs section?
What's the process?

**Reply by axhamre** [reply]
That sounds reasonable. If not, some examples here would do as well I think.

**Comment by benoitgrelard** [comment]
Honestly I am not sure there is value in adding styled-components as the API is similar to Stitches (or any CSS-in-JS libraries for that matter). The intention for these examples is to show different flavours of CSS are possible.

We already have to maintain 3 different versions of the same demos, I'm not sure we'd want to support more than that, especially given the fact it wouldn't add much value.

**Comment by abdulrahimiliasu** [comment]
@benoitgrelard  In that case we can create a separate site hosted on netlify or vercel that contains only `styled-components` examples, then u can point it out in the Get Started with styling section for those interested, I'm sure plenty find it a hassle to change `stitches` to `styled-components` styling.

**Reply by axhamre** [reply]
I agree, that would be great.

**Reply by ryanleichty** [reply]
Before the Tailwind examples were added to the site, someone created and maintained this page separately. Something along this line is probably the ideal path forward for styled components https://tailwindcss-radix.vercel.app/
